### PR TITLE
Add Trident Parts Design Logo and correct Typos in README files

### PR DESCRIPTION
**Pull request by GadgetAngel**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

/printer_mods

#### How was it tested? 

A Voron User on discord requested someone design the Trident Parts Design Logo, so I created this Cricut Logo file and then they requested the .stl files so they can print out the logo in plastic. I then created the .stl files in Fusion 360.  They printed it out and I did not hear back from them complaining of any issues so I assume everything is working correctly.

I also updated the README files to describe the contents of each directory and added a URL link for the directory paths so the user can just click and be taken to the appropriate directory.

#### Any background context you want to provide?

This add the Trident Parts Design Logo to the rest of my Logo collection.

#### Screenshots (if appropriate)

Trident_DLogo
Trident_DLogo_250_Rear_Panel stl
Trident_DLogo_CAD stl
Trident_DLogo_WOHexOutline2jpg

### CODE REVIEW

**Review by cbc02009 — CHANGES_REQUESTED** [review]
Can you remove the .vscode folder?

**Comment by GadgetAngel** [comment]
> Can you remove the .vscode folder?

I will try to remove it now

**Comment by GadgetAngel** [comment]
> Can you remove the .vscode folder?

Removed

**Review by cbc02009 — CHANGES_REQUESTED** [review]

File: .vscode/ltex.dictionary.en-US.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Voron
```
[the remark this answers is not in the document]
Comment (GadgetAngel) [reply .vscode/ltex.dictionary.en-US.txt]: Ok let us try this again. I think I deleted it. So it should be removed now

[the remark this answers is not in the document]
Comment (GadgetAngel) [reply .vscode/ltex.dictionary.en-US.txt]: removed, I hope

**Review by cbc02009 — APPROVED** [review]

**Pull request merged**